foam_controlwrapper/foam_runner.py

```diff
@@ -27,24 +27,25 @@ def run(self):
             raise ValueError('Number of cores must be greater than zero')
         elif self.ncores == 1:
             cmd = self.solver + " -case "+self.case + " > " +\
-                os.path.join(self.case, 'log') + ' 2>&1'
-            subprocess.call(cmd, shell=True)
+                os.path.join(self.case, 'log')
+            subprocess.check_call(cmd, shell=True)
         else:
             # write and modify decomposeParDict file
 
             # decompose
             cmd = "decomposePar -force -case " + self.case + " > " +\
-                  os.path.join(self.case, 'log.decompose') + ' 2>&1'
-            subprocess.call(cmd, shell=True)
+                  os.path.join(self.case, 'log.decompose')
+
+            subprocess.check_call(cmd, shell=True)
             cmd = "mpirun -np " + str(self.ncores) + " " + self.solver +\
                   " -parallel -case " + self.case + " > " +\
-                  os.path.join(self.case, 'log') +\
-                  ' 2>&1'
-            subprocess.call(cmd, shell=True)
+                  os.path.join(self.case, 'log')
+            subprocess.check_call(cmd, shell=True)
```

Comment:
@khiltunen , i have changed the use of ```subprocess.call``to ```subprocess.checkcall```.  now, we can see where things are failing:

```
[0] --> FOAM FATAL ERROR: 
[0] No coarse levels created, either matrix too small for GAMG or nCellsInCoarsestLevel too large.
    Either choose another solver of reduce nCellsInCoarsestLevel.
[0] 
[0]     From function GAMGSolver::GAMGSolver(const word& fieldName,const lduMatrix& matrix,const FieldField<Field, scalar>& interfaceBouCoeffs,const FieldField<Field, scalar>& interfaceIntCoeffs,const lduInterfaceFieldPtrsList& interfaces,const dictionary& solverControls)
[0]     in file matrices/lduMatrix/solvers/GAMG/GAMGSolver.C at line 121.
[0] 
FOAM parallel run exiting

```

Do you have any idea for what the problem is here?